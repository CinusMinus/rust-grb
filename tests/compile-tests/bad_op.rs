use gurobi::*;
mod utils;

fn main() -> Result<()> {
  let _g = gag::Gag::stdout();
  create_model!(_g, m, x, y, z);
  c!(x + y + 2*z + 1);
  c!(2*z);
  c!(2/z);
  Ok(())
}
