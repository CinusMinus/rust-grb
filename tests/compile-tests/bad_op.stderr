error: expression should be a ==, >= or <= comparison
 --> $DIR/bad_op.rs:7:6
  |
7 |   c!(x + y + 2*z + 1);
  |      ^^^^^^^^^^^^^^^

error: expression should be a ==, >= or <= comparison
 --> $DIR/bad_op.rs:8:6
  |
8 |   c!(2*z);
  |      ^^^

error: expression should be a ==, >= or <= comparison
 --> $DIR/bad_op.rs:9:6
  |
9 |   c!(2/z);
  |      ^^^
